pom.xml

```diff
@@ -133,6 +133,11 @@
         <version>${slf4j.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
```

Comment:
As @robertandrewbain mentioned these managed dependencies can now be removed from the parent pom, as the child poms don't refer to it anymore.